.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,4 +10,16 @@ And the contributing guide: https://github.com/containous/traefik/blob/master/.g
 
 <!--
 Briefly describe the pull request in a few paragraphs.
--->
\ No newline at end of file
+-->
+
+<!--
+ PLEASE, DON'T REMOVE THIS CHECKLIST AND COMPLETE IT.
```

Comment:
Thinking more about it, I don't think the check boxes will serve us well:

- If the users ticks them off, it's still upon the maintainers to verify whether all points are addressed properly. Chances are they won't, at which point the regular code review process will kick in to make sure things gets addressed. So they make no sense for the user.
- If we say the check boxes are for us, then there's no guarantee the PR author (or someone else) will accidentally (un-)check them. Github does not keep record of who toggles a box (and when) either.

Consequently, the only way the points make sense to me is to dedicate them to the user and remove the check boxes. Everything else should be covered by the usual review process.